admin_guide/allocating_node_resources.adoc

```diff
@@ -101,6 +101,50 @@ status:
 ----
 ====
 
+[[system-resources-reported-by-kubelet]]
+== System resources reported by kubelet
+
+Since openshift 3.3 the kubelet reports system resources utilized by the container runtime and kubelet.
+To better aid your ability to configure *--system-reserved* and *--kube-reserved*,
+you can introspect the nodes resource usage using the kubelet summary api which is accessible at *<node>:<port>/stats/summary*.
+
+To access the API from localhost with kubelet listening on default port, you can run:
+
+----
+$ curl -k https://127.0.0.1:10250/stats/summary
```

Comment:
I see only:

```json
   "systemContainers": [
    {
     "name": "runtime",
     "startTime": "2016-08-12T13:56:27Z",
     "cpu": {
      "time": "2016-08-12T13:59:03Z",
      "usageNanoCores": 4128509,
      "usageCoreNanoSeconds": 21635282299
     },
     "memory": {
      "time": "2016-08-12T13:59:03Z",
      "usageBytes": 4157440,
      "workingSetBytes": 4153344,
      "rssBytes": 4067328,
      "pageFaults": 1387477,
      "majorPageFaults": 0
     },
     "rootfs": {
      "availableBytes": 57583702016,
      "capacityBytes": 63345520640
     },
     "logs": {
      "availableBytes": 57583702016,
      "capacityBytes": 63345520640
     },
     "userDefinedMetrics": null
    }
   ],
```

no ``kubelet``. @derekwaynecarr is that the correct behaviour?